### PR TITLE
Update transgui.uk

### PR DESCRIPTION
**Pull request by uniss2209**
Error correction

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Hi @uniss2209,

Thank you for your contribution, I'm sorry that I don't speak Українська, do you know anyone on GitHub who can help us verify the change? Thanks!

**Comment by PeterDaveHello** [comment]
and maybe the change in #1419 can be merged here, so that we can do it at once?

**Comment by uniss2209** [comment]
> and maybe the change in #1419 can be merged here, so that we can do it at once?

yes, they can be merged

File: lang/transgui.uk [hunk 1]
```diff
@@ -284,7 +284,7 @@ Move down=Перемістити вниз
 Move top=Перемістити в початок
 Move up queue=Вгору в черзі
 Move up=Перемістити вгору
-No updates have been found.~You are running the latest version of %s=Оновлення не знайдені.~У вас встановлена ​​найновіша версія %s
+No updates have been found.~You are running the latest version of %s=Оновлення не знайдені.~У вас встановлена найновіша версія %s
```
Comment (PeterDaveHello) [inline lang/transgui.uk:287]: Looks like the two lines are the same? 🤔

Comment (uniss2209) [reply lang/transgui.uk:287]: No. There were extra characters.
2023-04-30 17_48_37-HxD -  Untitled1

Comment (PeterDaveHello) [reply lang/transgui.uk:287]: hmmm... it's interesting that can't be identified in visual editor 😱

**Review by PeterDaveHello — APPROVED** [review]
Checked with GPT-4

**Pull request merged**